html/captive-portal/lib/captiveportal/PacketFence/Controller/Authenticate.pm

```diff
@@ -459,7 +459,9 @@ sub authenticationLogin : Private {
         my $mac       = $portalSession->clientMac;
         my $node_info = node_view($mac);
         my $username = $node_info->{'last_dot1x_username'};
-        if ($username =~ /^(.*)@/ || $username =~ /^[^\/]+\/(.*)$/ ) {
+        #Strip the username when using machine auth
+        #TODO Do we config stripping the username for machine and user auth
+        if ( $username =~ /^[^\/]+\/(.*)$/ ) {
             $username = $1;
         }
```

Comment:
We can probably add a warn in the code like : $logger->warn("Reuse Dot1x has been enabled on this portal, here the username from 802.1x connection ".$node_info->{'last_dot1x_username'}." , here the stripped username $username");